PCbuild/python.props

```diff
@@ -61,17 +61,38 @@
   <Import Project="$(ExternalProps)" Condition="$(ExternalProps) != '' and Exists('$(ExternalProps)')" />
 
   <PropertyGroup>
-    <sqlite3Dir Condition="$(sqlite3Dir) == ''">$(ExternalsDir)sqlite-3.35.5.0\</sqlite3Dir>
+    <sqlite3Dir>$(SQLITE3_DIR)</sqlite3Dir>
+    <sqlite3Dir Condition="$(sqlite3Dir) == ''">$(ExternalsDir)sqlite-3.45.1.0\</sqlite3Dir>
```

Comment:
Is this right?

Reply:
This only matters if it can't find the sqllite *stuff*, but I will update it.